pkg/rke/templates/flannel_v0.22.2.go

```diff
@@ -0,0 +1,248 @@
+package templates
+
+/*
+FlannelTemplateV0_22_2 is based on upstream flannel v0.22.2
+Source: https://raw.githubusercontent.com/flannel-io/flannel/master/Documentation/kube-flannel.yml
+Upstream Changelog:
+- added get verb for nodes in flannel ClusterRole
+- added clustercidrs resources list and watch permission for flannel ClusterRole.
+Rancher Changelog:
+- No new Rancher specific changes, same as FlannelTemplateV0_21_4
+*/
+
+const FlannelTemplateV0_22_2 = `
+{{- if eq .RBACConfig "rbac"}}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels:
+    k8s-app: flannel
+  name: flannel
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - nodes/status
+  verbs:
+  - patch
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - clustercidrs
+  verbs:
+  - list
+  - watch
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels:
+	k8s-app: flannel
+  name: flannel
```

Comment:
This needs to be fixed at multiple places:
```suggestion
metadata:
  labels:
    k8s-app: flannel
  name: flannel
```

Reply:
Updated the PR with suggested changes